#ifndef BCSCREENMOVE_H
#define BCSCREENMOVE_H

#include <QDialog>

namespace Ui {
class BCScreenMove;
}

class BCScreenMove : public QDialog
{
    Q_OBJECT

public:
    explicit BCScreenMove(QWidget *parent = 0);
    ~BCScreenMove();

private slots:
    void on_horizontalSlider_1_valueChanged(int value);

    void on_horizontalSlider_2_valueChanged(int value);

    void on_horizontalSlider_3_valueChanged(int value);

    void on_horizontalSlider_4_valueChanged(int value);

    void on_lineEdit_1_textChanged(const QString &arg1);

    void on_lineEdit_2_textChanged(const QString &arg1);

    void on_lineEdit_3_textChanged(const QString &arg1);

    void on_lineEdit_4_textChanged(const QString &arg1);

private:
    Ui::BCScreenMove *ui;
};

#endif // BCSCREENMOVE_H
